### ISSUE TITLE
Width and height are calculated wrong, when only one of both is given

### ISSUE THREAD

**Issue by mogoh** [django-cms/djangocms-picture#91]
Width and height are calculated wrong, when only height is set and width is calculated.

## Example

- To reproduce, just use a default django-cms installation, for example this one: https://github.com/mogoh/django_examples/tree/master/cms
- Upload a picture for example this one: ![test2](https://user-images.githubusercontent.com/580975/74256958-75c81700-4cf4-11ea-8732-62c1b3fbda14.png)
  It has a resolution of 200×100.
- Embed the picture in a page, setting height to e.g. 50.

The result is a picture with the name `test2.png__80x50_q85_subsampling-2.jpg` and the resolution of 40×40.

## Why is that so?

When height is given, but width is not, width is calculated [here](https://github.com/divio/djangocms-picture/blob/db9c4d49ae1b840666e5146949d536d67386e1fc/djangocms_picture/models.py#L261):

~~~
        elif not width and height:
            width = int(height * PICTURE_RATIO)
~~~

This would be fine, if `PICTURE_RATIO` would actually be the ratio.
But `PICTURE_RATIO`, as capital letters suggest, is not calculated, but a constant, as you can see [here](https://github.com/divio/djangocms-picture/blob/db9c4d49ae1b840666e5146949d536d67386e1fc/djangocms_picture/models.py#L53).
It should be used, when not width nor height is given.

50×1.6=80 witch is, why the name contains 80x50.
Somewhere else, I don't know where, the actual size is calculated witch results in 80÷2=40 (40x40).

This is basically the same if only width is given.
But in this case, height is calculated wrongly, witch results in a wrong filename, but not in wrong actual size, because the actual size must be calculated somewhere only by resolution and width.
I haven't searched so far.

Please fix this issue.

**Comment by FinalAngel** [comment]
Will be fixed by #98

**Comment by fsbraun** [comment]
IS fixed by #100
